src/cvodes/cvodes.c

```diff
@@ -8500,13 +8545,14 @@ static int cvQuadEwtSet(CVodeMem cv_mem, N_Vector qcur, N_Vector weightQ)
 
 static int cvQuadEwtSetSS(CVodeMem cv_mem, N_Vector qcur, N_Vector weightQ)
 {
-  N_VAbs(qcur, cv_mem->cv_tempvQ);
-  N_VScale(cv_mem->cv_reltolQ, cv_mem->cv_tempvQ, cv_mem->cv_tempvQ);
-  N_VAddConst(cv_mem->cv_tempvQ, cv_mem->cv_SabstolQ, cv_mem->cv_tempvQ);
+  SUNAssignSUNCTX(CV_SUNCTX);
+  SUNCheckCallLastErrNoRet(N_VAbs(qcur, cv_mem->cv_tempvQ));
+  SUNCheckCallLastErrNoRet(N_VScale(cv_mem->cv_reltolQ, cv_mem->cv_tempvQ, cv_mem->cv_tempvQ));
+  SUNCheckCallLastErrNoRet(N_VAddConst(cv_mem->cv_tempvQ, cv_mem->cv_SabstolQ, cv_mem->cv_tempvQ));
   if (cv_mem->cv_atolQmin0) {
     if (N_VMin(cv_mem->cv_tempvQ) <= ZERO) return(-1);
```

Comment:
As above, needs a `sunrealtype minval` declared at the top of the function, and then this should be changed to something like
```suggestion
    minval = SUNCheckCallLastErrNoRet(N_VMin(cv_mem->cv_tempvQ));
    if (minval <= ZERO) return(-1);
```